acra/configuring-maintaining/optimisations/acrastructs_vs_acrablocks.md

```diff
@@ -0,0 +1,27 @@
+---
+title: AcraStructs vs AcraBlocks
+weight: 3
+---
+
+# AcraStructs vs AcraBlocks
+
+There are two different "crypto containers" AcraServer can use to store encrypted value in.
+Then, these containers are stored in database and decrypted back by AcraServer when needed.
+
+History
+* `AcraStruct` is our "old" one, though we're not going to remove it any time soon.
```

Comment:
It's not an old one. It is one of two. AcraStructs main crypto container for client-side encryption for now. Applications can encrypt with ephemeral asymmetric keys on app side with AcraStructs, but cannot do this with AcraBlocks because of symmetric key encryption. But they can configure AcraServer to re-encrypt AcraStructs (that generated on app side) on AcraServer side. Because AcraStructs allow to be sure that data are encrypted when it is transmitted between app and AcraServer.

Reply:
Rewrote this file a lot